_targets.R

```diff
@@ -0,0 +1,35 @@
+library(targets)
+source("code.R")
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    download_data(out_filepath = "model_RMSEs.csv"),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
+    process_data(in_filepath = model_RMSEs_csv),
+  ),
+  # Create a plot
```

Comment:
Like with the downloaded data, let's move this plot output file into the appropriate phase folder (so, something like `3_visualize/out/figure_1`). You can also rename the figure if you'd like to use something more descriptive.